src/components/Container/__tests__/__snapshots__/index.js.snap

```diff
@@ -289,50 +124,7 @@ exports[`Container renders correctly with small variant 1`] = `
   margin-top: 40px;
 }
 
-.emotion-2.emotion-2 {
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-box-pack: justify;
-  -webkit-justify-content: space-between;
-  justify-content: space-between;
-}
-
-.emotion-3.emotion-3 {
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-align-items: center;
-  -webkit-box-align: center;
-  -ms-flex-align: center;
-  align-items: center;
-  -webkit-box-pack: start;
-  -ms-flex-pack: start;
-  -webkit-justify-content: flex-start;
-  justify-content: flex-start;
-  margin-bottom: 8px;
-}
-
-.emotion-5 {
-  color: #4a4f62;
-  font-weight: 400;
-  margin-bottom: 0;
-  margin-top: 0;
-  color: #151a2d;
-  font-size: 25px;
-  line-height: 25px;
-  margin-bottom: 0;
-}
-
-.emotion-5.emotion-5 {
-  margin-top: 0;
-  margin-bottom: 0;
-  margin-right: 16px;
-}
-
-.emotion-8 {
+.emotion-4 {
```

Comment:
@vincentaudebert  there is a lot of css delete in this test ?

Reply:
CSS changes are linked to the upgrade. They don't create an extra class to add the "custom css" CSS but integrate it directly into existing one

Reply:
Ah and added a if condition which is not displaying a block if we dont have all the params (to avoid warnings). Which is probably why so much changes on the file. I just checked and snapshots are up to date.